src/devtools/views/AuthorizationPage.html

```diff
@@ -1,3 +1,10 @@
+<!--<style>-->
```

Comment:
New auth panel looks awesome!
Last thing, please tidy up the file and remove commented out code.

Reply:
done